messageV2.go

```diff
@@ -0,0 +1,300 @@
+package pglogrepl
+
+import (
+	"encoding/binary"
+	"time"
+)
+
+type MessageDecoderV2 interface {
```

Comment:
Exported symbols must have comment, according to Golang lint requirement.

Reply:
Other MessageV2 symbols are all exported, not only this one, StreamStart, StreamStop, etc.